### DISCUSSION TITLE
Reuse `DTypeFloatTorch` in `surrogates.utils`

### DISCUSSION THREAD

**Discussion by rjavadi** [emdgroup/baybe#161] [Q&A] [answered]
Since `_DTYPE` and `DTypeFloatTorch` (in `baybe.utils.numerical`) are both alias of `torch.float64`, could we just import `DTypeFloatTorch` in `surrogates.utils`? I think we will be able to load `torch` lazily in `surrogates.utils`in this way.

**Comment by Scienfitz** [comment, marked as answer]
yes i think `_DTYPE` is possibly a remainder of a refactoring anyway and no dtype cosntants should be declared outside of `utils.numerical`

**Reply by rjavadi** [reply]
Okay then I use `DTypeFloatTorch`.
I have another question regarding `baybe.utils.numerical`. I think we should also load it lazily, since it needs `torch` and there's no way to load it lazily inside `baybe.utils.numerical`.

**Reply by Scienfitz** [reply]
would you be able to make a dedicated `baybe.utils.torch`? i think that goes best along the lines of creating a clean bubble with lazily loaded torch only utils

**Comment by Scienfitz** [comment]
closing as no longer relevant
